### ISSUE TITLE
npm run clean:all fails on first pull of repo

### ISSUE THREAD

**Issue by benloh** [theRAPTLab/meme#3]
In GitLab by @daveseah on Apr 1, 2019, 16:45

to reproduce, run `npm run clean:all` twice.

error: 
``` 
> meme-boilerplate@0.0.1 clean:all /Users/loh/dev/inq-seeds/boilerplate
> node meme clean --all

internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'shelljs'
    at Function.Module._resolveFilename 
    [...]
```

This is caused because `meme.js` requires `shell.js` when it's run as a script via `package.json`. However, since you just ran `npm run clean:all`, shelljs has been removed so `meme.js` throws the error.

**PROPOSED FIX**

Use `require.resolve()` to trap error, if trapping the regular `require()` doesn't work. Print a nice error message about needing to run `npm ci` first.

**Comment by benloh** [comment]
In GitLab by @daveseah on Apr 2, 2019, 03:26

Resolved in e335d79d by using regular try/catch block

**Comment by benloh** [comment]
In GitLab by @daveseah on Apr 2, 2019, 03:29

closed
